src/assets/scss/_gift-config.scss

```diff
@@ -1,3 +1,11 @@
+product-config-form {
+  .btn:focus, 
+  .btn:active:focus, 
+  input[type="radio"]:focus + .btn-radio.active {
+    outline: -webkit-focus-ring-color auto 1px;
```

Comment:
This looks really good! Would it work to use the same focus ring styles as inputs for consistency?

```scss
.form-control:focus {
  border-color: $colorCru-blueBright;
  outline: 0;
}
```

Reply:
The outer label might be dark gray, but the inner input appears to be blue.

<img width="868" height="179" alt="Screenshot 2025-10-16 at 2 17 53 PM" src="https://github.com/user-attachments/assets/86b3fd11-aec3-4fb0-95fc-1155c7993252" />